common/sgx/verifier.c

```diff
@@ -443,96 +444,114 @@ static oe_result_t _fill_with_known_claims(
             valid_until,
             sizeof(*valid_until)));
 
-        // SGX quote PCESVN
-        OE_CHECK(oe_sgx_add_claim(
-            &claims[claims_index++],
-            OE_CLAIM_SGX_PCE_SVN,
-            sizeof(OE_CLAIM_SGX_PCE_SVN),
-            &sgx_quote->pce_svn,
-            sizeof(uint16_t)));
-
         // quote QEID
+        uint8_t ueid[OE_UEID_SIZE] = {0};
+        ueid[0] = OE_UEID_TYPE_RAND;
+        memcpy(&ueid[1], sgx_quote->user_data, SGX_USERDATA_SIZE);
+
         OE_CHECK(oe_sgx_add_claim(
             &claims[claims_index++],
-            OE_CLAIM_SGX_QE_ID,
-            sizeof(OE_CLAIM_SGX_QE_ID),
-            sgx_quote->user_data,
-            SGX_USERDATA_SIZE));
+            OE_CLAIM_UE_ID,
+            sizeof(OE_CLAIM_UE_ID),
```

Comment:
```suggestion
            OE_CLAIM_UEID,
            sizeof(OE_CLAIM_UEID),
```
consistency with IETF RATS doc and lines 456 and 457